templates/advantage/index.html

```diff
@@ -226,318 +140,38 @@ <h2>Your enterprise subscriptions</h2>
-              <div class="p-code-copyable">
-                <input class="p-code-copyable__input" value="sudo ua enable cc-eal" readonly="readonly">
-                <button class="p-code-copyable__action" onclick="dataLayer.push({
-                  'event' : 'GAEvent',
-                  'eventCategory' : 'Advantage',
-                  'eventAction' : 'Copy enable command',
-                  'eventLabel' : 'Canonical Common Criteria  EAL2 Provisioning',
-                  'eventValue' : undefined
-                });">Copy to clipboard</button>
-              </div>
-
-              <p>To deactivate EAL2:</p>
-              <div class="p-code-copyable">
-                <input class="p-code-copyable__input" value="sudo ua disable cc-eal" readonly="readonly">
-                <button class="p-code-copyable__action" onclick="dataLayer.push({
-                  'event' : 'GAEvent',
-                  'eventCategory' : 'Advantage',
-                  'eventAction' : 'Copy disable command',
-                  'eventLabel' : 'Canonical Common Criteria  EAL2 Provisioning',
-                  'eventValue' : undefined
-                });">Copy to clipboard</button>
-              </div>
-            </span>
-          </span>
-          <div class="u-hide u-show--small">
-            <div class="p-card u-no-margin--bottom">
-              <h4>EAL2</h4>
-              <p>To activate EAL2:</p>
-              <div class="p-code-copyable">
-                <input class="p-code-copyable__input" value="sudo ua enable cc-eal" readonly="readonly">
-                <button class="p-code-copyable__action" onclick="dataLayer.push({
-                  'event' : 'GAEvent',
-                  'eventCategory' : 'Advantage',
-                  'eventAction' : 'Copy enable command',
-                  'eventLabel' : 'Canonical Common Criteria  EAL2 Provisioning',
-                  'eventValue' : undefined
-                });">Copy to clipboard</button>
-              </div>
-
-              <p>To deactivate EAL2:</p>
-              <div class="p-code-copyable">
-                <input class="p-code-copyable__input" value="sudo ua disable cc-eal" readonly="readonly">
-                <button class="p-code-copyable__action" onclick="dataLayer.push({
-                  'event' : 'GAEvent',
-                  'eventCategory' : 'Advantage',
-                  'eventAction' : 'Copy disable command',
-                  'eventLabel' : 'Canonical Common Criteria  EAL2 Provisioning',
-                  'eventValue' : undefined
-                });">Copy to clipboard</button>
-              </div>
-            </div>
-          </div>
-        </li>
-        {% endif %}
-      </ul>
+      {% if personal_account['entitlements']['livepatch'] %}
+      <div class="row">
+        <div class="col-1">Livepatch:</div>
+        <div class="col-6"><pre style="border: 0; padding: 0;"><code>sudo ua enable livepatch</pre></code></div>
+      </div>
```

Comment:
This is a table. Any reason not to use `<tr>` and `<td>`? And put the `attach` line in the same layout.

Reply:
Done